cloud/aws/courses/windows.yaml

```diff
@@ -4,10 +4,25 @@ master:
   type: "m4.xlarge"
   key_name: "training"
   security_group_ids:
-    - "sg-688b120f"
+    - "sg-f3b33794"
```

Comment:
Is this a breaking change as we transition?

Reply:
This changes brings the PEW Windows instances up to date with the same specs that are used for Windows instances in the PI course. AFAICT, this `windows.yaml` file probably wasn't used before because PEW was always hosted on Skytap?